k8s/overlays/coreso/prod/kustomization.yaml

```diff
@@ -159,5 +206,26 @@ images:
     newName: docker.io/farao/gridcapa-task-manager
     newTag: v1.13.0
```

Comment:
We need also to upgrade the version of task manager for cse also to have the same version in the prod environment